pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -103,6 +103,82 @@ func (db *DB) Migration() *migrate.Migration {
 					)`,
 				},
 			},
+			{
+				// some databases may have already this done, although the version may not match
+				Description: "Extending bandwidth_agreements table and drop bwusagetbl",
+				Version:     1,
```

Comment:
Maybe if it's 3 different versions, the code will be clearer... but it can also be a single func. Maybe we need something that applies a list of funcs or SQL-s combined?

We can solve this later.